src/FormField/ScopeBuilder.php

```diff
@@ -524,16 +543,16 @@ public static function conditionToQuery(Condition $condition): array
 
             $map = [
                 'LIKE' => [
-                    'equals',
-                    'begins with',
-                    'ends with',
-                    'contains'
+                    self::OPERATOR_EQUALS,
+                    self::OPERATOR_BEGINS,
+                    self::OPERATOR_ENDS,
+                    self::OPERATOR_CONTAINS,
                 ],
                 'NOT LIKE' => [
-                    'does not equal',
-                    'does not begin with',
-                    'does not end with',
-                    'does not contain'
+                    self::OPERATOR_DOESNOT_EQUAL,
+                    self::OPERATOR_DOESNOT_BEGIN,
+                    self::OPERATOR_DOESNOT_END,
+                    self::OPERATOR_DOESNOT_CONTAIN,
```

Comment:
`DOESNOT` sounds not ideal - is `NOT_CONTAINS` better?

Reply:
I tried it but I did not like it

Reply:
You marked it `protected` that I think it is ok... I do not like boths but have no better now :D